packages/template-retail-react-app/app/commerce-api/einstein.js

```diff
@@ -127,8 +127,14 @@ class EinsteinAPI {
             ...(body && {
                 body: JSON.stringify(body)
             })
+        }).catch(() => {
+            console.warn('Einstein request failed')
         })
 
+        if (!response?.ok) {
+            return
```

Comment:
Quick note: I was unsure what to return here. My personal preference would be null.

But it could also be an empty object. In that case the fallbacks further down would not be needed. 🤔

Reply:
I'm leaning towards returning an empty object because then the fallback will be just here rather than in 3 places (getRecommenders, getRecommendations, and getZoneRecommendations all need the same fallback).